mssql.go

```diff
@@ -906,6 +907,35 @@ func (s *Stmt) makeParam(val driver.Value) (res param, err error) {
 		return
 	}
 	switch val := val.(type) {
+	case int:
+		res.ti.TypeId = typeIntN
+		// Rather than guess if the caller intends to pass a 32bit int from a 64bit app based on the
```

Comment:
Is this going to result in a potential breaking change to existing clients?

Reply:
yeah for any type != Int64 , we will now pass a matching size int instead of a BigInt.

Reply:
it's not "breaking" in the sense of an interface change, but it means that possibly some type conversions in existing queries will change. Maybe some apps have TSQL queries that expect all integer parameters to be bigint and now the SQL runtime will have to coerce them on the server side.